src/highlight.js

```diff
@@ -0,0 +1,18 @@
+const cards = document.querySelectorAll(".number-card")
+let randomNumbersInSequence = randomNumbersHolder.sort((function(a, b){return a-b}))
```

Comment:
this is a perfect use-case for a fat arrow function

Reply:
do you mean the stuff in the brackets of .sort?